Garden-Service/src/Garden-Service.Infrastructure/Services/GardenService.cs

```diff
@@ -38,4 +38,34 @@ public class GardenService : IGardenService {
 		};
 		return deleteGardenResponse;
 	}
+
+	public async Task<EditGardenResponse> EditGardenAsync(EditGardenRequest request) {
+		EditGardenResponse editGardenResponse;
+		var garden = _gardenContext.Gardens.SingleOrDefault(g => g.GardenId == request.GardenId);
+		if (garden != null) {
```

Comment:
Bitte nutze die C#9 Herangehensweise für das NullReference-Checking. `is null` oder `is not null`. Diese funktioniert unabhängig von den Operatorüberladungen, welche dazwischenfunken können

Reply:
wurde ausgestauscht